samples/openthread/coap_client_multiprotocol_ble/README.rst

```diff
@@ -0,0 +1,170 @@
+     When the sample application is ready, the LED 1 starts blinking.
+
+#. Set up nRF Toolbox by completing the following steps:
+
+   .. tabs::
+
+      .. tab:: 1. Start UART
+
+         Tap :guilabel:`UART` to open the UART application in nRF Toolbox.
+
+         .. figure:: images/nrftoolbox_uart_default.png
+            :alt: UART application in nRF Toolbox
+
+            UART application in nRF Toolbox
+
+      .. tab:: 2. Configure commands
+
+         Configure the UART commands by completing the following steps:
+
+         1. Tap the :guilabel:`EDIT` button in the top right corner of the application.
+            The button configuration window appears.
+         #. Create the active application buttons by completing the following steps:
+
+            a. Bind the top left button to the ``u`` command, with EOL set to LF and an icon of your choice.
+               For this testing procedure, the :guilabel:`>` icon is used.
+            #. Bind the top middle button to the ``m`` command, with EOL set to LF and an icon of your choice.
+               For this testing procedure, the play button icon is used.
+            #. Bind the top left button to the ``p`` command, with EOL set to LF and an icon of your choice.
+               For this testing procedure, the settings gear icon is used.
+
+            .. figure:: images/nrftoolbox_uart_settings.png
+               :alt: Configuring buttons in nRF Toolbox - UART application
+
+               Configuring buttons in the UART application of nRF Toolbox
+
+         #. Tap the :guilabel:`DONE` button in the top right corner of the application.
+
+      .. tab:: 3. Connect to device
+
+         Tap :guilabel:`CONNECT` and select the ``NUS_CoAP_client`` device from the list of devices.
+
+         .. figure:: images/nrftoolbox_uart_connected.png
+            :alt: nRF Toolbox - UART application view after establishing connection
+
+            The UART application of nRF Toolbox after establishing the connection
+
+         .. note::
+            Observe that LED 2 on your CoAP Multiprotocol Client node is solid, which indicates that the Bluetooth connection is established.
+   ..
+
+#. In nRF Toolbox, press the middle button to control LED 4 on all CoAP server nodes.
+#. Pair a client with a server by completing the following steps:
+
+   a. Press Button 4 on a server node to enable pairing.
+   #. In nRF Toolbox, press the left button to pair the two nodes.
+
+#. In nRF Toolbox, press the right button to control LED 4 on the paired server node.
+   You can also control CoAP servers directly by pressing Button 1, Button 2, or Button 4 on the client development kit.
+
+Sample output
+==============
+
+The sample logging output can be observed through a serial port.
+For more details, see :ref:`putty`.
+
+Dependencies
+************
+
+This sample uses the following |NCS| libraries:
+
+* :ref:`nRF Bluetooth LE Controller <nrfxlib:ble_controller>`
```

Comment:
I wonder since you moved some of the code to libs shouldn't you mention it here?

Reply:
Good point